code/modules/mob/mob_helpers.dm

```diff
@@ -586,4 +600,19 @@ proc/is_blind(A)
 
 	return ..()
 
-
+/mob/proc/get_sex()
+	return gender
```

Comment:
Seem redundant

Reply:
Entire mob folder will be replaced in baymed no point to fix it now